BD-3935 Add behavior for Canvases that are stopped and reactivated (#9096)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/messaging_fundamentals/about_statuses.md

---
nav_title: Statuses
article_title: Campaign and Canvas statuses
page_order: 1
tool: Campaigns
description: "Learn about statuses for campaigns and Canvases and how to use them in the dashboard."
---

> # Statuses

Reviewer: ```suggestion
# Campaign and Canvas statuses
```

## Changing the status

To change the status of a campaign or Canvas, select the <i class="fas fa-ellipsis-vertical"></i> menu, then choose a status.

![A list of Canvases in the Braze dashboard, with the menu open for one of the Canvases.]({% image_buster /assets/img/messaging_fundamentals/change-status.png %})

## Available statuses

These are the available statuses for campaigns and Canvases:

| Status | Description |
| --- | --- |
| Active | Active campaigns and Canvases are in the process of sending. By default, you'll see active campaigns and Canvases on the respective pages. |
| Draft | Drafts of campaigns and Canvases are saved but not launched. To continue editing and begin sending, you can select the draft by going to **Messaging** in the Braze dashboard and selecting **Canvas** or **Campaigns**. |
| Archived | Archived campaigns and Canvases are messages that are no longer being sent. These campaigns and Canvases are also removed from the statistic graphs on the [**Home**]({{site.baseurl}}/user_guide/analytics/dashboard/home_dashboard) and [**Revenue**]({{site.baseurl}}/user_guide/analytics/reporting/revenue_report) pages.|

> | Stopped | Stopped campaigns and Canvases are paused, but you can still edit them. To resume, select the <i class="fas fa-ellipsis-vertical"></i> menu, then **Resume**. For more information, see [Stopped Canvas behavior](#stopped-canvas-behavior). |

Author: ```suggestion
| Stopped | Stopped campaigns and Canvases are paused, but you can still edit them. To resume, select the <i class="fas fa-ellipsis-vertical"></i> menu, then **Resume**. For more information, refer to [Stopped Canvas behavior](#stopped-canvas-behavior). |
```

| Idle | When a campaign or Canvas is no longer sending messages, Braze will assign it an idle status to help sort and manage your list of campaigns and Canvases. You can view which campaigns or Canvases will be automatically stopped and the associated stop date. |
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

### Stopped Canvas behavior {#stopped-canvas-behavior}

When a Canvas is stopped, the following occurs:

> - **Scheduled messages:** Your scheduled messages won't be sent, regardless of a user's place in the Canvas. This also includes users who were queued due to rate limiting.

Author: ```suggestion
- **Scheduled messages:** Your scheduled messages won't be sent, regardless of a user's place in the Canvas. This also includes users who were queued because of rate limiting.
```

> - **Scheduled messages:** Your scheduled messages won't be sent to users, regardless of their place in the Canvas. This also includes users who were queued due to rate limiting.

Reviewer: ```suggestion
- **Scheduled messages:** Your scheduled messages won't be sent, regardless of a user's place in the Canvas. This also includes users who were queued due to rate limiting.
```
